native/acter/src/api/attachments.rs

```diff
@@ -375,3 +239,149 @@ impl AttachmentsManager {
         self.client.subscribe(self.inner.update_key())
     }
 }
+
+impl MsgContentDraft {
+    async fn into_attachment_content(
+        &self,
+        client: SdkClient,
+        room: Room,
+    ) -> Result<Option<AttachmentContent>> {
+        match &self {
+            MsgContentDraft::TextPlain { .. } => Ok(None),
+            MsgContentDraft::TextMarkdown { .. } => Ok(None),
+            MsgContentDraft::Image { source, info } => {
+                let info = info.as_ref().expect("image info needed");
+                let mimetype = info.mimetype.clone().expect("mimetype needed");
+                let content_type = mimetype.parse::<mime::Mime>()?;
+                let path = PathBuf::from(source);
+                let mut image_content = if room.is_encrypted().await? {
```

Comment:
According to the timeline docs, send_attachment already transparently takes care of the encryption part now... if that is why client and room are necessary, then we probably want to refactor that as ot isn't needed for timeline and should probably work similar on our spaces, too.. (not inside the conversion fn)

Reply:
But `send_attachment()` doesn't expose `ImageMessageEventContent`.
We need to wrap it `AttachmentContent::Image()`.

Reply:
But the fn is returning an `AttachmentContent` ... so, that is what we need to give to `timeline.send_attachment`. I don't understand the problem...